kolibri/plugins/coach/assets/test/showPageActions.spec.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env mocha */
+const sinon = require('sinon');
+const kolibri = require('kolibri');
+
+// Add fake Resources to kolibri mock. This needs to be done before loading tested module
+kolibri.resources = {
+  ChannelResource: {
+    getCollection: sinon.stub(),
+  },
+  ClassroomResource: {
+    getModel: sinon.stub(),
+  },
+  ContentNodeResource: {},
+  ExamResource: {
+    getCollection: sinon.stub(),
+  },
+  LearnerGroupResource: {
+    getCollection: sinon.stub(),
+  },
+};
+
+const examActions = require('../src/state/actions/exam');
```

Comment:
I can't, because the `kolibri` object needs to be modified before requiring the 'exam.js' module. Otherwise when 'exam.js' itself requires 'kolibri', it errors out trying access things from `kolibri`, which is mocked as `{}`.